### PR TITLE
Migrate to `Weight::from_parts`

### PR DESCRIPTION
**Pull request by ggwpez**
Use `Weight::from_parts` instead of `from_ref_time` and `from_proof_size`. One stop migration script: [migrate-from-parts.py](https://github.com/ggwpez/substrate-scripts/blob/master/migrate-from-parts.py).

Companion for https://github.com/paritytech/substrate/pull/13475  
Analogous to https://github.com/paritytech/polkadot/pull/6794

### CODE REVIEW

**Review by KiChjang — APPROVED** [review]

**Review by sam0x17 — APPROVED** [review]
Screenshot 2023-03-02 at 2 10 00 PM 
🥲

**Comment by ggwpez** [comment]
bot merge

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2476441

File: parachains/runtimes/assets/statemine/src/weights/cumulus_pallet_xcmp_queue.rs [hunk 1]
```diff
@@ -54,8 +54,8 @@ impl<T: frame_system::Config> cumulus_pallet_xcmp_queue::WeightInfo for WeightIn
 		//  Measured:  `76`
 		//  Estimated: `571`
 		// Minimum execution time: 4_956 nanoseconds.
-		Weight::from_ref_time(5_108_000)
-			.saturating_add(Weight::from_proof_size(571))
+		Weight::from_parts(5_108_000, 0)
+			.saturating_add(Weight::from_parts(0, 571))
```
Comment (gavofyork) [inline parachains/runtimes/assets/statemine/src/weights/cumulus_pallet_xcmp_queue.rs:58]:
```suggestion
		Weight::from_parts(5_108_000, 571)
```

So many more instances of this... I think we need to reweigh everything with the new weights code?

Comment (ggwpez) [reply parachains/runtimes/assets/statemine/src/weights/cumulus_pallet_xcmp_queue.rs:58]: Yes, re-weighting happens before each release. It takes 6-8 hours AFAIK and so to get it merged sooner I replaced it in the code.

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**